tests/benchmark_expr_templates.cpp

```diff
@@ -62,7 +62,7 @@ static void BM_mixed_expr_MFEM(benchmark::State& state)
   MPI_Barrier(MPI_COMM_WORLD);
 
   // Number of rows is the argument that varies
-  const int rows  = state.range(0);
+  const int rows  = static_cast<int>(state.range(0));
```

Comment:
is it worth it to just define `sample_vectors` to take a `size_t` or whatever `state.range(0)` is, rather than static casting everything to `int`?

Reply:
I'll get this merged but leave this comment as an issue.